test/specs/first.js

```diff
@@ -0,0 +1,137 @@
+//import { expect, browser, $ } from '@wdio/globals'
+
+
+describe('My Login application', () => {
+
+
+    it('Perform Login', async () => {
+        await browser.url(`https://www.saucedemo.com`)
+
+
+        // выполняю логин на страницу
+        await $('#user-name').addValue("standard_user")
+        await $('#password').addValue("secret_sauce")
+        await $('#login-button').click()
+
+
+   
+        await expect($("//span[text()='Products']")).toBeDisplayed() // проверяем наличие тайтла Products
+        await expect($('#shopping_cart_container')).toBeDisplayed()  // проверяем наличие иконки Cart
+
+
+
+
+            // проверяем что количество айтэмов больше 1
+
+
+            let elem = await $$("//div[@class='inventory_list'] /div");
```

Comment:
You can type:
```suggestion
            expect(await $$("//div[@class='inventory_list'] /div").length).to.be.greaterThen(1);
```
or something like this. Note: my code example above might contain typos - it's just to show the idea